### ISSUE TITLE
Gateway solution possible?

### ISSUE THREAD

**Issue by Njur** [jotyGill/openpyn-nordvpn#138]
Hi first of all this is a very excellent job you have done with this script.
What I would like and hope you can help is to be able to use it as a gateway, so I have clients connecting to the machine with your VPN script installed.
Is it possible you can help with this?

**Comment by jotyGill** [comment]
Thank you.
Have you tried openpyn without "-f"? does that work?

**Comment by Njur** [comment]
Yes it works, but would like to use it as a gateway and with killswitch functionality.

**Comment by jotyGill** [comment]
Check this out https://github.com/mrahmadt/SmartGW
